bootstrap/bukkit/src/main/java/org/geysermc/platform/bukkit/GeyserBukkitPlugin.java

```diff
@@ -73,7 +74,18 @@ public void onEnable() {
         this.connector = GeyserConnector.start(PlatformType.BUKKIT, this);
 
         this.geyserCommandManager = new GeyserBukkitCommandManager(this, connector);
-        this.geyserWorldManager = new GeyserBukkitWorldManager();
+
+        boolean isViaVersion = false;
+        if (Bukkit.getPluginManager().getPlugin("ViaVersion") != null) {
+            // TODO: Update if ViaVersion updates or Minecraft updates
+            if (!Via.getAPI().getVersion().equals("3.0.0-SNAPSHOT") && !Bukkit.getServer().getVersion().contains("1.15.2")) {
```

Comment:
Might be good to include a version check here in addition to the Via check that way so if they are running 1.13+ with a different version of ViaVersion, they don't get this error. 

On another note, are we only supporting snapshot versions of ViaVersion and not releases? I think it'd be better to target releases here, rather than snapshots. As far as I'm aware, the abstraction branch only includes version for the nether update, and IMO I don't see much of a reason to support those versions as there is no release for 1.16 yet. If for some reason future updates are only on the abstraction branch or the latest release moves on to 3.0.0, then it'd be better to change this around.

Reply:
Yeah, this is mostly to anticipate future updates only being on the abstraction branch.

Reply:
That's true indeed, VB only works with 3.0.0. abstraction will also soon be merged into master and all development is done in the abstraction branch until then